websites/G/Genshin Impact Map/presence.ts

```diff
@@ -0,0 +1,163 @@
+const presence = new Presence({
+		clientId: "973729201104511007"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+interface Map {
```

Comment:
Map is a node class, you shouldn't be shadowing global definitions so you should probably name this something different

Reply:
This should be addressed